### PR TITLE
Better support for Persian and other right to left languages

### PR DESCRIPTION
**Pull request by a5hk**
Added support for Persian language. Other right to left languages should also look better now.

**Before Modifications:**
before-mod

**After Modifications:**
after-mod

### CODE REVIEW

**Comment by kinson** [comment]
Thanks for doing this 🙂 I will review this in the next day or two!

**Comment by doomspork** [comment]
@a5hk this is _awesome_! Could you take a look at running `mix format` locally to clean up that CI failure? Thank you @kinson for jumping on the review

**Comment by a5hk** [comment]
Sure! Done. @doomspork

File: lib/school_house_web/views/html_helpers.ex [hunk 1]
```diff
@@ -62,6 +62,17 @@ defmodule SchoolHouseWeb.HtmlHelpers do
     )
   end
 
+  # mod -> module
+  # asgn -> assigns
+  def load_locale_styles(mod, asgn) do
+    sanitized_locale = String.replace(current_locale(), "-", "_")
```
Comment (kinson) [inline lib/school_house_web/views/html_helpers.ex:68]: Thanks for handling locales with `-` here! 🙌🏻

**Review by kinson — APPROVED** [review]
🎉 thanks for your patience with my review, this is awesome 🥳

**Pull request merged**